test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,27 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        //empty list
+        assertFalse( false);
```

Comment:
We are testing the `Utils.isAnyNull` method here. `assertFalse(false)` will always be correct. Instead, you should be asserting `assertFalse(Utils.isAnyNull())`.